bundles/binding/org.openhab.binding.weather/src/main/java/org/openhab/binding/weather/internal/common/LocationConfig.java

```diff
@@ -24,7 +24,7 @@
     private Double latitude;
     private Double longitude;
     private String woeid;
-    private Integer updateInterval;
+    private Integer updateInterval = 240;
```

Comment:
something like this?

    private static final int DEFAULT_UPDATE_INTERVAL = 240;

Not sure I understand what this is for, but it's easy enough...